client/src/test/java/org/spine3/base/StringifiersShould.java

```diff
@@ -337,11 +343,16 @@ public void convert_list_of_integers_to_string() {
     @Test
     public void convert_string_to_list_of_integers() {
         final String stringToConvert = "1|2|3|4|5";
-        final List<Integer> convertedList =
-                new Stringifiers.ListStringifier<>(Integer.class, "\\|").reverse()
-                                                                        .convert(stringToConvert);
-        assertNotNull(convertedList);
-        assertEquals(5, convertedList.size());
+        final String delimiter = "\\|";
+        final List<Integer> actualList =
+                new Stringifiers.ListStringifier<>(Integer.class,
+                                                   delimiter).reverse()
+                                                             .convert(stringToConvert);
+        assertNotNull(actualList);
+        assertEquals(5, actualList.size());
```

Comment:
Let's remove this line. It does nothing (since there are more checks), but adds another magic constant.